lib/secure_headers/headers/public_key_pins.rb

```diff
@@ -1,23 +1,37 @@
 module SecureHeaders
-  class PublicKeyPinsBuildError < StandardError; end
+  class PublicKeyPinsConfigError < StandardError; end
   class PublicKeyPins < Header
-    module Constants
-      HPKP_HEADER_NAME = "Public-Key-Pins"
-      ENV_KEY = 'secure_headers.public_key_pins'
-      HASH_ALGORITHMS = [:sha256]
-      DIRECTIVES = [:max_age]
-      CONFIG_KEY = :hpkp
-    end
+    HEADER_NAME = "Public-Key-Pins".freeze
+    HASH_ALGORITHMS = [:sha256].freeze
+    DIRECTIVES = [:max_age].freeze
+    CONFIG_KEY = :hpkp
```

Comment:
Since we're using these in the `request.env`, it seems like they should be more unique to avoid conflicts. Maybe `SECURE_HEADERS_HPKP`, for example?